### ISSUE TITLE
Check Extras 7.34 Returns PASS on ACCESS DENIED. Should it return FAIL?

### ISSUE THREAD

**Issue by jonathanbglass** [prowler-cloud/prowler#327]
I ran this with no access to s3:getbucket*, and received the following output. I'd propose that an S3 Access Denied message should not generate a PASS. I'd recommend it generate a FAIL so analysts know to look into that bucket and policy a little deeper.

`7.34 | PASS | Not Scored | Extra | [extra734] Check if S3 buckets have default encryption (SSE) enabled and policy to enforce it (Not Scored) (Not part of CIS benchmark) | Bucket XXX has SSE enabled with algorithm An error occurred (AccessDenied) when calling the   GetBucketEncryption operation: Access Denied but no S3 enforcing policy found!`

**Comment by toniblyx** [comment]
if not FAIL at least something different to PASS.

**Comment by toniblyx** [comment]
hey @jonathanbglass can you test the latest version of the check? It was updated 2 weeks ago.

**Comment by jonathanbglass** [comment]
Now it reports a PASS and a FAIL. 

```
Laptop:prowler jonathan$ git status
On branch master
Your branch is up to date with 'origin/master'.

$ aws --profile profile s3api get-bucket-encryption --bucket BUCKET_NAME

An error occurred (AccessDenied) when calling the GetBucketEncryption operation: Access Denied

$ ~/GitHub/prowler/prowler -c extra734 -p profile -M csv | grep BUCKET_NAME

Generating "," delimited report on stdout for profile profile, account xxxxxxxxxxxx
0.0 Show report generation info
7.34 [extra734] Check if S3 buckets have default encryption (SSE) enabled or use a bucket policy to enforce it (Not Scored) (Not part of CIS benchmark)
profile,xxxxxxxxxxxx,us-east-1,7.34,PASS,Not Scored,Extra,[extra734] Check if S3 buckets have default encryption (SSE) enabled or use a bucket policy to enforce it (Not Scored) (Not part of CIS benchmark),Bucket BUCKET_NAME has default encryption enabled with algorithm An error occurred (AccessDenied) when calling the GetBucketEncryption operation: Access Denied
profile,xxxxxxxxxxxx,us-east-1,7.34,FAIL,Not Scored,Extra,[extra734] Check if S3 buckets have default encryption (SSE) enabled or use a bucket policy to enforce it (Not Scored) (Not part of CIS benchmark),Bucket BUCKET_NAME does not enforce encryption!
```

**Comment by toniblyx** [comment]
Hi @jonathanbglass did you find a solution for this issue? I can handle the error to avoid that but in essence it is a permissions issue, not actually Prowler issue, if Prowler doesn't have access to a resource it will fail, it needs to get/read resources.

**Comment by gabrielsoltz** [comment]
Fixed here: https://github.com/toniblyx/prowler/pull/339

**Comment by toniblyx** [comment]
As @gabrielsoltz said, it is fixed in #339 and here https://github.com/toniblyx/prowler/commit/eecb272f93f7126cb9a7c9856d9db8af736b597b
